include/bluetooth/bluetooth.h

```diff
@@ -166,6 +166,40 @@ int bt_set_name(const char *name);
  */
 const char *bt_get_name(void);
 
+/**
+ * @brief Get local Bluetooth appearance
+ *
+ * Bluetooth Appearance is a description of the external appearance of a device
+ * in terms of an Appearance Value.
+ *
+ * @see
+ *  https://specificationrefs.bluetooth.com/assigned-values/Appearance%20Values.pdf
+ *
+ * @returns
+ *  Appearance Value of local Bluetooth host.
```

Comment:
Just a matter of taste/readability. It does not affect the Doxygen output. I'm not sure what's best. What is your preference?

Reply:
My preference is to follow the current style of the file as per the codestyle  guidelines :) 

My other (personal) preference is that I actually kind of like this approach, but if done, should be done for the entire file. Besides making separation of several `@param` easier, it will also make it easier to fit on one line for longer parameter names.

Reply:
Would it be possible to get a tool to automate Doxygen formatting?

Reply:
> 

AFAIK the doxygen formatting style in the public headers are quite different at this point, so that may be hard to enforce, but not impossible. Many IDEs already come with doxygen formatting support (or extension to it), so I expect that there are tools that verify the style too.